code/src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from'react'
+
+export const Footer = () => {
+  return (
+    <>
+      <h3>Approved for public release.</h3>
+      <h3>The electronic version is the official approved document.</h3>
+      <h3>(IoSS) does not take any responsebility in case of death.</h3>
```

Comment:
This is outside of the requirements in my opinion, but one suggestion is to create some sort of container for these h3-elements to be in (a div for instance). Now they are pressed to the bottom of the page which isn’t that great in my opinion - the h3-elements needs some space to breathe as well 😅 

Also in the mobile view the h3-elements now hover over the text of the form since the footer is set to position fixed.. If you had a container for the footer this could be fixed with for example setting a background color 🙂